src/phoenix/db/models.py

```diff
@@ -516,6 +516,7 @@ class Experiment(Base):
     )
     name: Mapped[str]
     description: Mapped[Optional[str]]
+    repetitions: Mapped[int]
```

Comment:
can default to 1

Reply:
bit confused here @anticorrelator I don't see changes in the migrations file?